src/components/LandEnsPage/LandEnsPage.container.ts

```diff
@@ -0,0 +1,21 @@
+import { connect } from 'react-redux'
+import { RootState } from 'modules/common/types'
+import { MapStateProps, MapDispatchProps, MapDispatch } from './LandEnsPage.types'
+import LandEditPage from './LandEnsPage'
+import { getENSRequest, GET_ENS_REQUEST, SET_ENS_REQUEST, setENSRequest } from 'modules/ens/actions'
+import { isLoadingType } from 'decentraland-dapps/dist/modules/loading/selectors';
+import { getState, getLoading, getError } from 'modules/ens/selectors';
+import { Land } from 'modules/land/types';
+
+const mapState = (state: RootState): MapStateProps => ({
+  error: getError(state),
+  ens: getState(state),
```

Comment:
We typically use the `data` as the value we pass to components instead of having to supply the entiry state (which also contains things like loading props and errors).
Maybe we can change it here and the components that receive it down the chain